internal/stream/docs.go

```diff
@@ -15,7 +15,7 @@ func Spec() docs.FieldSpecs {
 		}
 	}
 
-	defaultOutput := map[string]any{"inproc": ""}
+	defaultOutput := map[string]any{"reject": "default reject"}
```

Comment:
Can we have a bit better error message?

```suggestion
	defaultOutput := map[string]any{"reject": "message rejected by default because an output is not configured"}
```